extension/extend-admin-console-spi/README.md

```diff
@@ -0,0 +1,36 @@
+## Extending the admin ui with SPI
+
+This example shows how you can extend the admin ui by implementing the SPI.
+You can either add a tab to an existing page or have a complete section in the menu.
+By implementing either:
+
+  * `org.keycloak.services.ui.extend.UiPageProvider` for a complete section
+  * `org.keycloak.services.ui.extend.UiTabProvider` for a tab
+
+For both options you need to describe the fields you want on the page by using the `org.keycloak.provider.ProviderConfigProperty`
+To specify where the tab will appear in the ui, you can add a path valid paths can be found in the `routes.ts` files.
+
+This example contains an implementation of both these `AdminUiPage` is a todo section that is added.
+And `ThemeUiTab` is an extra tab in the realm setting that let's you edit the realm attributes.
+
+### Running
+
+Run this example by compiling with maven:
+
+```bash
+$ mvn install
+```
+
+Then copy the generated `extend-admin-ui-1.0-SNAPSHOT.jar` into your <keycloak-server>/provider folder
+
+### Screenshots
+
+As you can see from the `getPath` from the `ThemeUiTab` example this page can be found when you navigate to "realm settings"
```

Comment:
This assumes knowledge about `getPath` being automatically detected and linked into the _Realm settings_. This may not be necessarily obvious. Is there any reference that would clarify this functionality?

```suggestion
To illustrate how `getPath` method is used, check the `ThemeUiTab` class in this quickstart. The ThemeUi page can now be found when you navigate to _Realm settings_.
```